### ISSUE TITLE
the possibility of changing one of ROIs volumes.

### ISSUE THREAD

**Issue by arasheb2001** [qurit/rt-utils#50]
Hi guys. 
I am a medical researcher and working on DICOM files in my current project.
I wonder to know is it possible to be able to uniformly extend or shorten one of ROIs in rt-struct DICOM file?  
Uniformly extending or shortening PTV(Planning Target Volume) in rt-struct DICOM file and obtaining the overlap volume between this PTV and surrounding Organ at risks (OAR) lead us to calculate OVH (Overlap Volume Histogram) which is a useful feature feeding machine learning models.

**Comment by asim-shrestha** [comment]
Hi @arasheb2001, we currently have no plans for this functionality. What you can do at the moment is extract the ROI data through RT-Utils, run your own custom pipeline for your desired transformation, and update the ROI through RT-Utils. If you would like, you can also make these changes yourself and submit a pull request so that we can include this feature within the library.

**Comment by Lauchitacrypto** [comment]
Hi @arasheb2001 , could you figure it out? I am working on a similar project and would be a big help if you could tell me some.
Thanks!

**Comment by arasheb2001** [comment]
> Hi [@arasheb2001](https://github.com/arasheb2001) , could you figure it out? I am working on a similar project and would be a big help if you could tell me some. Thanks!

Hi , could you tell me exactly which part?

**Comment by Lauchitacrypto** [comment]
> > Hi [@arasheb2001](https://github.com/arasheb2001) , could you figure it out? I am working on a similar project and would be a big help if you could tell me some. Thanks!
> 
> Hi , could you tell me exactly which part?

Yes, I want to crop the PTV in every point where it overlaps with an OAR. At the moment, I am working with Prostate cases so I am trying to crop it from the Bladder and Rectum. So I was wondering if you had a function already developed or if you know how to address the issue. @arasheb2001
